src/libertem/web/state.py

```diff
@@ -43,9 +46,22 @@ def get_context(self) -> Context:
     async def set_executor(self, executor: JobExecutor, params):
         if self.executor is not None:
             await self.executor.close()
+            self.executor = None
+        self._set_executor(executor, params)
+
+    def _set_executor(self, executor: JobExecutor, params):
+        if self.executor is not None:
+            self.executor.ensure_sync().close()
         self.executor = executor
         self.cluster_params = params
         self.context = Context(executor=executor.ensure_sync())
+        try:
+            # Exposing the scheduler address allows use of
+            # libertem-server to spin up a LT-compatible
+            # persistent dask cluster
+            log.info(f'Scheduler at {self.context.executor.client.scheduler.address}')
```

Comment:
Just a BTW., not to be solved in this PR, distributed by default seems to listen on `0.0.0.0` - we might want to change this to something a bit more explicit and secure.